src/config/config_file/rtxrc.rs

```diff
@@ -309,6 +309,32 @@ impl RTXFile {
             .unwrap();
         plugin_aliases[from] = toml_edit::value(to);
     }
+
+    pub fn remove_alias(&mut self, plugin: &str, from: &str) {
+        let doc = self.get_or_create_edit();
+        if doc.contains_table("alias") {
+            let aliases = doc
+                .as_table_mut()
+                .entry("alias")
+                .or_insert(toml_edit::table())
+                .as_table_mut()
+                .unwrap();
+            if aliases.contains_key(plugin) {
+                let plugin_aliases = aliases
+                    .entry(plugin)
+                    .or_insert(toml_edit::table())
+                    .as_table_mut()
+                    .unwrap();
+                plugin_aliases.remove(from);
+                if plugin_aliases.is_empty() {
+                    aliases.remove(plugin);
+                }
+            }
+            if aliases.is_empty() {
+                doc.as_table_mut().remove("alias");
+            }
+        }
+    }
```

Comment:
This can be cleaned up (possibly even more than this, but this was the best I could do). I tested this so I'm sure it's good.

```suggestion
    pub fn remove_alias(&mut self, plugin: &str, from: &str) {
        let doc = self.get_or_create_edit();
        if let Some(aliases) = doc.get_mut("alias").and_then(|v| v.as_table_mut()) {
            if let Some(plugin_aliases) = aliases.get_mut(plugin).and_then(|v| v.as_table_mut()) {
                plugin_aliases.remove(from);
                if plugin_aliases.is_empty() {
                    aliases.remove(plugin);
                }
            }
            if aliases.is_empty() {
                doc.as_table_mut().remove("alias");
            }
        }
    }
```

Reply:
and btw I realize you probably just copy+pasted similar code that I had written in this file. This file is some of the oldest code in the codebase and has some of the worst code since I'm still new at rust.